src/site/content/en/blog/intersectionobserver/index.md

```diff
@@ -0,0 +1,147 @@
+---
+layout: post
+title: IntersectionObserver's coming into view
+subhead: IntersectionObservers let you know when an observed element enters or exits the browser's viewport.
+authors:
+  - surma
+description: IntersectionObservers let you know when an observed element enters or exits the browser's viewport.
+date: 2016-04-20 
+updated: 2022-08-15
+tags:
+  - blog
+  - javascript
+  - performance
+---
+
+{% Aside %}
+`IntersectionObserver` V2 has landed in Chromium-based browsers. [Read up on the updates](/intersectionobserver-v2/) for how that API improves on the first version of `IntersectionObserver`.
+{% endAside %}
```

Comment:
This look good, @tunetheweb?

Reply:
Can't say I know enough about it, but seems it's not a complete replacement](https://web.dev/intersectionobserver-v2/#:~:text=For%20that%20reason%2C%20Intersection%20Observer%C2%A0v2%20is%20not%20intended%20to%20be%20used%20broadly%20in%20the%20way%20that%20Intersection%20Observer%C2%A0v1%20is.) so yeah I think that's good and generic enough.